rotkehlchen/api/rest.py

```diff
@@ -4842,35 +4842,22 @@ def export_history_events(
             # maintain insertion order without storing extra info
             headers.update(dict.fromkeys(serialized_event))
 
-        if directory_path is None:  # on download
-            with tempfile.TemporaryDirectory(ignore_cleanup_errors=True) as temp_dir:  # needed on windows, see https://tinyurl.com/tmp-win-err  # noqa: E501
-                file_path = Path(temp_dir) / FILENAME_HISTORY_EVENTS_CSV
-                try:
-                    dict_to_csv_file(
-                        path=file_path,
-                        dictionary_list=serialized_history_events,
-                        csv_delimiter=settings.csv_export_delimiter,
-                        headers=headers.keys(),
-                    )
-                    register_post_download_cleanup(file_path)
-                    return send_file(
-                        path_or_file=file_path,
-                        mimetype='text/csv',
-                        as_attachment=True,
-                        download_name=FILENAME_HISTORY_EVENTS_CSV,
-                    )
-                except (CSVWriteError, PermissionError) as e:
-                    return wrap_in_fail_result(
-                        message=str(e),
-                        status_code=HTTPStatus.CONFLICT,
-                    )
-                except FileNotFoundError:
-                    return wrap_in_fail_result(
-                        message='No file was found',
-                        status_code=HTTPStatus.NOT_FOUND,
-                    )
+        try:
+            if directory_path is None:  # file will be downloaded later
```

Comment:
fill in the method of download

```suggestion
            if directory_path is None:  # file will be downloaded later via XXX endpoint
```